CMakeLists.txt

```diff
@@ -47,7 +47,7 @@ include(FetchContent)
 FetchContent_Declare(
   repo-cutlass
   GIT_REPOSITORY https://github.com/NVIDIA/cutlass.git
-  GIT_TAG        cc85b64cf676c45f98a17e3a47c0aafcf817f088
+  GIT_TAG        6f47420213f757831fae65c686aa471749fa8d60
```

Comment:
Can cutlass be upgrade to 3.2? There are some msvc build issues before 3.2

Reply:
let's disable flash attention 2 If it is hard to support on windows platform